frontend/src/metabase/setup/components/LanguageStep.jsx

```diff
@@ -67,6 +67,7 @@ export default class LanguageStep extends React.Component {
               ).map(([code, name]) => (
                 <li
                   key={code}
+                  aria-language={code}
```

Comment:
Works for me :)

I'd just use `data-testid` here.

Reply:
On the second thought, we should definitely NOT use `aria-language` here. Let's leave `aria` attributes for a11y. This can possibly confuse screen readers even more.

`data-testid` was made for this specific purpose in mind.

Reply:
You got it 👍